### PR TITLE
Rename --cloud-aws-subnet-id to --cloud-aws-subnet

### LINKED ISSUE #868: AWS subnet & security groups: ID vs Name
- [x] https://github.com/iterative/terraform-provider-iterative/pull/364
- [ ] #869

### PR DESCRIPTION
**Pull request by 0x2b3bfa0**
Closes #868

### CODE REVIEW

File: bin/cml/runner.js [hunk 1]
```diff
@@ -532,11 +533,16 @@ exports.builder = (yargs) =>
...
+      cloudAwsSubnet: {
         type: 'string',
         default: '',
         description: 'Specifies the subnet to use within AWS'
       },
+      cloudAwsSubnetId: {
+        type: 'string',
+        default: '',
+        hidden: true
+      },
```
Comment (casperdcl) [inline bin/cml/runner.js:545]: should be aliased

**Review by casperdcl — CHANGES_REQUESTED** [review]

File: bin/cml/runner.js (hunk 1 above)
Comment (0x2b3bfa0) [reply bin/cml/runner.js:545]: Yes, but aliases can't be hidden. At least, I haven't found yet how to hide them.

Comment (casperdcl) [reply bin/cml/runner.js:545]: I'd rather let the alias show. Simplifies the code a lot. We could always leave out the alias in the online docs.

Comment (0x2b3bfa0) [reply bin/cml/runner.js]: 👌🏼

**Review by casperdcl — APPROVED** [review]

**Pull request merged**